scripts/pack-packages.sh

```diff
@@ -0,0 +1,30 @@
+# This script is used in the "npm pack" step in our CI build pipelines.
```

Comment:
Might be nice to run in a more strict mode, and make it easier to invoke manually with bash.

```suggestion
#!/bun/bash
set -eux -o pipefail

# This script is used in the "npm pack" step in our CI build pipelines.
```

Reply:
For my education:

> -x     The  shell shall write to standard error a trace for each command after it expands the command and
       before it executes it. It is unspecified whether the command that turns tracing off is traced.

Does that mean every command gets "echoed" before execution?

Reply:
yes, which is nice for reading CI logs.

Reply:
It can be jarring at first, but it's really helpful for debugging. Makes the logs look a lot more like an interactive shell session.